eZ/Bundle/EzPublishCoreBundle/Resources/config/services.yml

```diff
@@ -38,20 +38,55 @@ services:
         class: "%ezpublish.siteaccess.class%"
         arguments: ["%ezpublish.siteaccess.default.name%", 'uninitialized']
 
-    ezpublish.config.resolver.core:
```

Comment:
There are still some references to that service in the codebase.
Please also add a bc note that this service has been removed.